sigopt/run_params.py

```diff
@@ -30,7 +35,7 @@ def setdefaults(self, *args, **kwds):
     unset_keys = set(tmp) - set(items)
     update = {key: tmp[key] for key in unset_keys}
     items.update(update)
-    _get(self, "__run_context").set_parameters(update)
+    _get(self, "__run_context").set_parameters(items)
```

Comment:
why this change? this uploads all of the values every time `setdefaults` is used. The existing behavior only modifies the values that are changed

Reply:
ok, I will change it back.